src/opnsense/www/js/opnsense_widget_manager.js

```diff
@@ -322,6 +322,7 @@ class WidgetManager  {
                 message: $content,
                 buttons: [{
                     label: this.gettext.add,
+                    cssClass: 'btn-primary',
```

Comment:
in theory this is nicer but primary color for header and button doesn't match which is annoying. we could also pivot to "warning" dialogs here? not sure if "yellow" is better.. at least better than "red" (danger)

Reply:
I removed the primary hint in the modals so this is now just a consistency fix for the `confirm()` replacement.